apis/python/src/tiledbsoma/io/spatial/_util.py

```diff
@@ -2,21 +2,59 @@
 #
 # Licensed under the MIT License.
 
+from __future__ import annotations
+
 from pathlib import Path
-from typing import Tuple, Union
+from typing import Any, Union
 
 import h5py
+import numpy as np
+import numpy.typing as npt
+import pyarrow as pa
+import pyarrow.compute as pacomp
+from typing_extensions import Self
 
 from ..._exception import SOMAError
 
 
+def _expand_ptr(indptr: npt.NDArray, nval: int) -> npt.NDArray:
```

Comment:
All else being equal I'd prefer `_expand_ptr` unless p-t-r per se is a specific term of art here